<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>light_path_selector</class>
 <widget class="QWidget" name="light_path_selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>124</width>
    <height>19</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="cmb_scope_channel"/>
   </item>
   <item>
    <widget class="QComboBox" name="cmb_light_path"/>
   </item>
   <item>
    <widget class="condenser_config" name="wdg_condenser_config" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>condenser_config</class>
   <extends>QWidget</extends>
   <header>condenser_config.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
